src/main/resources/application.yaml

```diff
@@ -2,6 +2,12 @@ server:
   servlet:
     context-path: /api
 
+management:
+  endpoints:
+    web:
+      exposure:
```

Comment:
I think it would be good to setup the url prefix for actuator url, something like:
management:
 endpoints:
 web:
 base-path: /management

So the urls would be http://{ak-domain-url}/management/health or http://{ak-domain-url}/management/metrics.

Reply:
Done 🎉